tests/meta/workflow/conftest.py

```diff
@@ -113,6 +116,39 @@ def microsalt_qc_fail_lims_project() -> str:
     return "ACC11111_qc_fail"
 
 
+@pytest.fixture
+def metrics_file_failing_qc(
+    microsalt_qc_fail_run_dir_path: Path,
+    microsalt_qc_fail_lims_project: str,
+    tmp_path: Path,
+) -> Path:
+    """Return a metrics file that fails QC with corresponding samples in the database."""
+    metrics_path = Path(microsalt_qc_fail_run_dir_path, f"{microsalt_qc_fail_lims_project}.json")
+    temp_metrics_path = Path(tmp_path, metrics_path.name)
+    shutil.copy(metrics_path, temp_metrics_path)
+    return temp_metrics_path
+
+
+@pytest.fixture
+def metrics_file_passing_qc(
+    microsalt_qc_pass_run_dir_path: Path,
+    microsalt_qc_pass_lims_project: str,
+    tmp_path: Path,
+) -> Path:
+    """Return a metrics file that fails QC with corresponding samples in the database."""
+    metrics_path = Path(microsalt_qc_pass_run_dir_path, f"{microsalt_qc_pass_lims_project}.json")
+    temp_metrics_path = Path(tmp_path, metrics_path.name)
+    shutil.copy(metrics_path, temp_metrics_path)
+    return temp_metrics_path
+
+
+@pytest.fixture
+def valid_microsalt_metrics_file(
+    microsalt_qc_fail_run_dir_path: Path, microsalt_qc_fail_lims_project: str
+) -> Path:
+    return Path(microsalt_qc_fail_run_dir_path, f"{microsalt_qc_fail_lims_project}.json")
+
```

Comment:
Slightly confused by the optimistically named fixture but the pessimistically named path components.

Reply:
it is used to test the metrics parser. Renamed it to just `microsalt_metrics_file` 👍